art.taunoerik.tauno-serial-plotter.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
+Encoding=UTF-8
+Type=Application
+Name=Tauno-serial-plotter
+Comment=Simple serial plotter
+TryExec=tauno-serial-plotter
+Exec=tauno-serial-plotter
+Icon=art.taunoerik.tauno-serial-plotter
+MimeType=image/x-svg;
```

Comment:
just to clarify, the application open SVG files?

Reply:
That is a mistake, the application does not open SVG files.